### PR TITLE
Add ScaleInControl fields to GCE Autoscalers

### PR DESCRIPTION
**Pull request by upodroid**
Fixes: https://github.com/hashicorp/terraform-provider-google/issues/7621

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [X] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [X] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [X] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
compute: added `autoscaling_policy.0.scale_in_control` fields to `google_compute_autoscaler`
```
```release-note:enhancement
compute: added `autoscaling_policy.0.scale_in_control` fields to `google_compute_region_autoscaler`
```

### CODE REVIEW

File: products/compute/api.yaml [hunk 1]
```diff
@@ -341,6 +341,36 @@ objects:
...
+              - !ruby/object:Api::Type::Integer
+                name: 'fixed'
+                description: |
+                  Specifies a fixed number of VM instances. This must be a positive
+                  integer.
+              - !ruby/object:Api::Type::Integer
+                name: 'percent'
+                description: |
+                  Specifies a percentage of instances between 0 to 100%, inclusive.
+                  For example, specify 80 for 80%.
```
Comment (ScottSuarez) [inline products/compute/api.yaml:368]: Why not have calculated as output only? By design?

File: third_party/terraform/tests/resource_compute_autoscaler_test.go.erb [hunk 2]
```diff
@@ -239,6 +264,31 @@ resource "google_compute_autoscaler" "foobar" {
...
+  autoscaling_policy {
+    max_replicas    = 10
+    min_replicas    = 1
+    cooldown_period = 60
+    cpu_utilization {
+      target = 0.5
+    }
+    scale_in_control {
+      max_scaled_in_replicas {
+        percent = 80
```
Comment (ScottSuarez) [inline third_party/terraform/tests/resource_compute_autoscaler_test.go.erb:287]: Add a test for fixed scenario

**Review by ScottSuarez — CHANGES_REQUESTED** [review]
Some suggestions... Also can we add/update an example where we use one of these new fields in the documentation.

File: products/compute/api.yaml (hunk 1 above)
Comment (upodroid) [reply products/compute/api.yaml:368]: I took pointers from scaleDownControls which similar to the new fields. https://github.com/GoogleCloudPlatform/magic-modules/pull/3701/files

File: third_party/terraform/tests/resource_compute_autoscaler_test.go.erb (hunk 2 above)
Comment (upodroid) [reply third_party/terraform/tests/resource_compute_autoscaler_test.go.erb:287]: done

**Comment by upodroid** [comment]
@ScottSuarez Bump

**Review by ScottSuarez — APPROVED** [review]

**Comment by ScottSuarez** [comment]
Adding @c2thorn for a second pass !

**Review by c2thorn — APPROVED** [review]
LGTM

**Pull request merged**